### PR TITLE
Add RisingStack as sponsor

### PR DESCRIPTION
**Pull request by Shadowbeetle**
Requesting merge to add RisingStack as a sponsor after the official announcement next Tuesday

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -151,6 +151,7 @@ <h2>Sponsors</h2>
         <div class="sponsors-wrapper">
             <div class="sponsor">
                 <span class="vertical-helper"></span><a href="https://developer.ibm.com/technologies/node-js/"><img class="small-img" src="./assets/images/sponsors/IBM-logo.svg" alt="IBM Logo"></a>
+                <span class="vertical-helper"></span><a href="https://risingstack.com"><img class="small-img" src="./assets/images/sponsors/RisingStack-logo.svg" alt="RisingStack Logo"></a>
```
Comment (tomleah) [inline index.html]: Please move this into a new <div class="sponsor">

Comment (gdams) [reply index.html]: fixed

File: index.html [hunk 2]
```diff
@@ -152,6 +152,9 @@ <h2>Sponsors</h2>
             <div class="sponsor">
                 <span class="vertical-helper"></span><a href="https://developer.ibm.com/technologies/node-js/"><img class="small-img" src="./assets/images/sponsors/IBM-logo.svg" alt="IBM Logo"></a>
             </div>
+            <div class="sponsor">
+                <span class="vertical-helper"></span><a href="https://risingstack.com"><img class="small-img" src="./assets/images/sponsors/RisingStack-logo.svg" alt="RisingStack Logo"></a>
```
Comment (tomleah) [inline index.html]: nit: `small-img` class is not needed here

**Comment by tomleah** [comment]
@Shadowbeetle could you sign the [CLA] (https://cla-assistant.io/CloudNativeJS/cloudnativejs.io?pullRequest=58) so this can be merged Tuesday

**Comment by Shadowbeetle** [comment]
> @Shadowbeetle could you sign the [CLA](https://cla-assistant.io/CloudNativeJS/cloudnativejs.io?pullRequest=58) so this can be merged Tuesday

Signed, is there anything else to be done before Tuesday?

**Pull request merged**